portfolio/src/main/webapp/index.html

```diff
@@ -27,10 +26,22 @@
                 <div id="greeting-container"></div>
                  <button class="button" onclick="addRandomIntro()">I am a...</button>
                  <div id="intro-container"></div>
+                <div class="bottom">
+                    <p>Zoe represents "life" and that best describes my personality. 
```

Comment:
For long paragraphs, please nest the content between the braces:

```
<p>
    Zoe represents "life"...
    ...
    ...
</p>
```

This looks a bit more readable right? :)